app/celery/scheduled_tasks.py

```diff
@@ -375,3 +380,68 @@ def delete_old_records_from_events_table():
     current_app.logger.info(f"Deleted {deleted_count} historical events from before {delete_events_before}.")
 
     db.session.commit()
+
+
+@notify_celery.task(name="zendesk-new-email-branding-report")
+def zendesk_new_email_branding_report():
+    previous_weekday = datetime.today().date() - timedelta(days=1)
+
+    # If yesterday is a Saturday or Sunday, adjust back to the Friday
+    if previous_weekday.isoweekday() in {6, 7}:
+        previous_weekday -= timedelta(days=(previous_weekday.isoweekday() - 5))
+
+    new_email_brands = (
+        EmailBranding.query.join(Organisation, isouter=True).filter(EmailBranding.created_at >= previous_weekday).all()
+    )
+
+    current_app.logger.info(f"{len(new_email_brands)} new email brands to review since {previous_weekday}.")
+
+    if not new_email_brands:
+        return
+
+    brands_by_organisation = defaultdict(list)
+    brands_with_no_organisation = []
+    for new_brand in new_email_brands:
+        if not new_brand.organisations:
+            brands_with_no_organisation.append(new_brand)
+
+        else:
+            for organisation in new_brand.organisations:
+                brands_by_organisation[organisation].append(new_brand)
+
+    template = jinja2.Template(
+        textwrap.dedent(
+            """
+            <p>There are new email brands to review since {{ yesterday }}.</p><hr>
+            {%- for org, brands in brands_by_organisation.items() -%}
+                <p><a href="{{ domain }}/organisations/{{ org.id }}/">{{ org.name }}</a>:</p><ul>
+                {%- for brand in brands -%}
+                    <li><a href="{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
+                {%- endfor -%}
+                </ul><hr>
+            {%- endfor -%}
+            {%- if brands_with_no_organisation -%}
+                <p>Unassociated brands:</p><ul>
+                {%- for brand in brands_with_no_organisation -%}
+                    <li><a href="{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
+                {%- endfor -%}
+                </ul>
+            {%- endif -%}
+            """
+        ).strip()
+    )
+    message = template.render(
+        domain=current_app.config["ADMIN_BASE_URL"],
+        yesterday=previous_weekday.strftime("%A %-d %B %Y"),
+        brands_by_organisation=brands_by_organisation,
+        brands_with_no_organisation=brands_with_no_organisation,
+    )
+
+    ticket = NotifySupportTicket(
+        subject="Review new email brandings",
+        message=message,
+        ticket_type=NotifySupportTicket.TYPE_INCIDENT,
```

Comment:
Is `TYPE_INCIDENT` appropriate here? Might [`TYPE_TASK`](https://github.com/alphagov/notifications-utils/blob/main/notifications_utils/clients/zendesk/zendesk_client.py#L50) be better?

Reply:
I had it as type_task originally but it was suggested we rarely use that and incident is more common for us.

Reply:
Have switched to TYPE_TASK on second thought, it really does just feel more appropriate...